transformations/whitespace_perturbation/transformation.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def whitespace(text, remove_prob=0.1, add_prob=0.05, seed=0):
+    random.seed(seed)
+    newtext = ''
+    for char in text:
+        random_num = random.random()
+        if char.isspace() and random_num < remove_prob:
+            continue
+        newtext += char
+        if (not char.isspace()) and random_num < add_prob:
+            newtext += ' '
+
+    return newtext
+
+
+class WhitespacePerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
```

Comment:
Consider adding more languages that are whitespace separated beyond just English.

Reply:
I added 4 more languages (`es`, `de`, `fr`, `it`) to the list, and added a discussion about which other languages won’t work in which cases to `README.md`.

Reply:
Oh, just saw this reply! You can have a look at https://github.com/GEM-benchmark/NL-Augmenter/pull/84/files/c9f1e4bdf8fbdacce706c3a6000a42455019c535..c38dd6535fc18b107b3f3bfe924cc370a323df89 as an example